### ISSUE TITLE
Distinguish external 2FA methods from built-in ones

### ISSUE THREAD

**Issue by ApCoder123** [2factorauth/twofactorauth#5930]
There are several sites listed for which certain 2FA methods are only available through a 3rd party provider (e.g. Duo)
https://github.com/2factorauth/twofactorauth/blob/6bfc7c8beebd37a1d533bfa982ab7e8e577ba053/entries/p/perimeter81.com.json#L16
https://github.com/2factorauth/twofactorauth/blob/d3e48e0ea561ae4d28b73bd4359af3f14f9fe20e/entries/1/1password.com.json#L10

It would be nice to be able to visually distinguish between built-in and 3rd-party 2FA methods, potentially through the icons. E.g.
![image](https://user-images.githubusercontent.com/20560218/124815015-9b476780-df5e-11eb-8856-70aba8ffa544.png)
rather than just having an exception text, as for non-Duo users who may be looking for 2FA methods supported, it could be frustrating to see a method listed that they can't actually use.

It could potentially be a useful feature to have on the site, although it could add clutter to the table?

**Comment by person1234-ux** [comment]
Would it be possible to add the ability to filter out duo 2fa when searching websites.

**Comment by Carlgo11** [comment]
@person1234-ux yes that is possible. Use `2fa:duo` in the search field on the site and it will show any sites supporting duo that we know of.
